packages/lighthouse-plugin-amp/README.md

```diff
@@ -0,0 +1,11 @@
+# Lighthouse AMP plugin
+
+[![npm version](https://badge.fury.io/js/%40ampproject%2Ftoolbox-cors.svg)](https://badge.fury.io/js/%40ampproject%2Ftoolbox-cors)
+
+AMP plugin for [Lighthouse](https://github.com/GoogleChrome/lighthouse).
+
+## To run
+
+1. Install `lighthouse` v5 or later.
+2. Install the plugin as a (peer) dependency, parallel to `lighthouse`.
+3. Run `npx -p lighthouse lighthouse https://example.com --plugins=lighthouse-plugin-amp --view`
```

Comment:
Can you please add all commands to run and install the plugin. I'm also curious: why do you need to install lighthouse and the plugin if it's run via `npx`?

Reply:
Suggestion: use `https://amp.dev` instead of `https://example.com`

Reply:
> Can you please add all commands to run and install the plugin. 
done.

>I'm also curious: why do you need to install lighthouse and the plugin if it's run via `npx`?

Pls see https://github.com/GoogleChrome/lighthouse/pull/9653. @paulirish is working on improving this and once fixed I'll make the change here as well.